mobile/contacts.go

```diff
@@ -70,8 +72,17 @@ func (m *Mobile) ContactThreads(id string) (string, error) {
 }
 
 // SearchContacts calls core SearchContacts
-func (m *Mobile) SearchContacts(query *pb.ContactQuery, options *pb.QueryOptions, cb Callback) (func(), error) {
```

Comment:
I made these byte slices because you'll already have the types from re-generating the JS proto files. You can look at `addThreadFiles` for a ref, it does the same.